setup.py

```diff
@@ -24,6 +25,12 @@
 VERSION = open('VERSION', 'r').read().strip()
 VIRTUAL_ENV = hasattr(sys, 'real_prefix')
 
+python_version = platform.python_version()
```

Comment:
`platform.python_version()` returns a string, and the comparison to another "version" string is not really fail proof.

I recomment you use something like:

```python
import sys
if sys.version_info[:2] == (2, 6)
...
```